ui/src/views/actions/thunks.ts

```diff
@@ -112,8 +115,49 @@ export const getViewForTimeMachine = (
         view,
       })
     )
+  } catch (error) {
+    console.error(error)
+    dispatch(notify(copy.getViewFailed(error.message)))
+    dispatch(setView(cellID, RemoteDataState.Error))
+  }
+}
+
+export const setQueryResultsByQueryID = (queryID: string) => (
+  dispatch,
+  getState: GetState
+): Promise<void> => {
+  try {
+    const state = getState()
+    const files = state.queryCache.queryResultsByQueryID[hashCode(queryID)]
```

Comment:
You are hashing queryID before sending to this function. so you shouldn't be hashing here.

Reply:
Hmmm, I'm not sure if i am, but I definitely should be since it's a little misleading at this point